test/clean-local-registry-tag.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+docker pull busybox
+
```

Comment:
For sanity check can you exit the script if $1 is `docker.io` or `quay.io`?

Reply:
Good call.